src/bridges/mean/MeanBridge.sol

```diff
@@ -45,6 +56,37 @@ contract MeanBridge is BridgeBase, Ownable2Step {
     // Note: we need to be able to receive ETH to deposit as WETH
     receive() external payable {}
 
+    /**
+     * @notice Registers wrappers internally, so that they can be referenced by an index instead of the full address
+     * @dev Anyone can call this method, it's not permissioned
+     * @param _wrappers The wrappers to register
+     */
+    function registerWrappers(address[] calldata _wrappers) external {
+        for (uint i; i < _wrappers.length; ) {
+            address _wrapper = _wrappers[i];
+            if (!DCA_HUB.allowedTokens(_wrapper)) {
+                revert MeanErrorLib.TokenNotAllowed(_wrapper);
+            } 
+            // Note: we could check that the address is indeed a wrapper, but we don't think it's necessary
+            // We check that the addresses are wrappers when we use them, and we have enough slots to add 
+            // non-wrapper tokens. So we just don't check it here and save gas
+            // Also, we know that we can only support 2**16 tokens on the registry, but we are not checking that
+            // here. We won't ever support that many tokens on our contract
+            bool _added = tokenWrapperRegistry.add(_wrapper);
+            if (!_added) {
+                revert MeanErrorLib.TokenAlreadyRegistered(_wrapper);
+            }
+
+            _maxApprove(IERC20(_wrapper), address(DCA_HUB));
+            _maxApprove(IERC20(_wrapper), address(TRANSFORMER_REGISTRY));
+
+            unchecked{
+                ++i;
+            }
+        }
+        emit NewWrappersSupported(_wrappers);
+    }    
```

Comment:
But it can't get to 65k ids, because:
- Only supported tokens can be added
- Tokens can only be added once

Since we won't support DCA-ing on 65k tokens anytime soon (we currently support only 6), I think we are safe from a DoS